src/main/java/seedu/address/logic/parser/AttendanceMarkingCommandParser.java

```diff
@@ -53,7 +54,12 @@ public AttendanceMarkingCommand parse(String args) throws ParseException {
                     MESSAGE_INVALID_COMMAND_FORMAT, MarkAttendanceCommand.MESSAGE_USAGE));
```

Comment:
Since we are going to loop through the telegramKeywords and run each of them through the parseTelegram function, and since that function already checks for empty string, lines 50 to 55 might be redundant.

Also, I think the error message displayed from the parseTelegram check for empty strings provides more context than the generic MESSAGE_INVALID_COMMAND_FORMAT error message.

If you want, you could remove lines 50-55

Reply:
Thanks for the review, removed the lines already.